### PR TITLE
Add some fractribution information

### PR DESCRIPTION
**Pull request by rlh1994**
- Clarify that while we say channel, you can technically define it to be whatever you like
- Add some content to explain how to use custom contexts for the conversion source 

The second part may need some additional info but I'm not super clear on what that table actually needs to have.

### CODE REVIEW

File: docs/modeling-your-data/modeling-your-data-with-dbt/dbt-models/dbt-fractribution-data-model/index.md [hunk 1]
```diff
@@ -28,6 +28,11 @@ The Fractribution package is dependent on the `snowplow_web_page_views` model cr
 ​
 The Snowplow Fractribution dbt package is a tool to perform Attribution Modeling. The name itself comes from `Fractional attribution`, which allows you to attribute the value of a conversion to one or more channels depending on the conversion pathway. As a result, it becomes possible to determine the revenue per channel, as well as ROAS (Return On Ad Spend - the amount of revenue that is earned for every dollar spent on advertising) once you have cost data for each marketing channel.
 ​
+:::tip
+
+While the throughout this package we use the term _channel_, it is entirely possible to define this in terms of something else e.g. a campaign.
```
Comment (agnessnowplow) [inline docs/modeling-your-data/modeling-your-data-with-dbt/dbt-models/dbt-fractribution-data-model/index.md:33]: I'd add that they can configure the `channel_spend` and `channel_classification` macro to do so.

File: docs/modeling-your-data/modeling-your-data-with-dbt/dbt-quickstart/fractribution/index.md [hunk 2]
```diff
@@ -88,8 +88,43 @@ vars:
 
 :::
 
+<details>
+<summary>Modifying the conversions source</summary>
+
+By default the `snowplow__conversions_source` is your atomic events table. In most cases this is likely to be what you want to use, however you may wish to use the in-built conversions modeling as part of our [web package](/docs/modeling-your-data/modeling-your-data-with-dbt/dbt-models/dbt-web-data-model/index.md), by setting `snowplow__conversions_source` to `"{{ ref(snowplow_joined_events_table') }}"`. Alternatively, if you are using Redshift/Postgres you may wish to include additional fields from a Self-Describing Event, or an Entity. 
+
+To do this, you should create a new model in your project, e.g. `models/snowplow/snowplow_joined_events_table.sql` which should have something like the following content. For more information about dealing with duplicates, make sure to see our [deduplication docs](/docs/modeling-your-data/modeling-your-data-with-dbt/dbt-advanced-usage/dbt-duplicates/index.md).
```
[the remark this answers is not in the document]
Comment (zloff) [reply docs/modeling-your-data/modeling-your-data-with-dbt/dbt-quickstart/fractribution/index.md:96]: The conversions source table really doesn't need many columns - https://github.com/snowplow/dbt-snowplow-fractribution/blob/main/models/snowplow_fractribution_conversions_by_customer_id.sql 
those mentioned plus the ones used in conversions_clause and conversion_value

File: docs/modeling-your-data/modeling-your-data-with-dbt/dbt-quickstart/fractribution/index.md [hunk 3]
```diff
@@ -88,8 +88,43 @@ vars:
 
 :::
 
+<details>
+<summary>Modifying the conversions source</summary>
+
+By default the `snowplow__conversions_source` is your atomic events table. In most cases this is likely to be what you want to use, however you may wish to use the in-built conversions modeling as part of our [web package](/docs/modeling-your-data/modeling-your-data-with-dbt/dbt-models/dbt-web-data-model/index.md), by setting `snowplow__conversions_source` to `"{{ ref(snowplow_joined_events_table') }}"`. Alternatively, if you are using Redshift/Postgres you may wish to include additional fields from a Self-Describing Event, or an Entity.
```
[the remark this answers is not in the document]
Comment (rlh1994) [reply docs/modeling-your-data/modeling-your-data-with-dbt/dbt-quickstart/fractribution/index.md:94]: Yeah this shouldn't be that table ref, I mucked up the copy paste 😅

File: docs/modeling-your-data/modeling-your-data-with-dbt/dbt-quickstart/fractribution/index.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (rlh1994) [reply docs/modeling-your-data/modeling-your-data-with-dbt/dbt-quickstart/fractribution/index.md:96]: Yeah I've just included all columns out of ease, the user could pull through less if they wanted.

**Review by agnessnowplow — APPROVED** [review]
LGTM

**Review by zloff — APPROVED** [review]
Nice, looks good!

**Pull request merged**